docs/resources/llama-stack-spec.html

```diff
@@ -2448,18 +2449,19 @@
                                 "$ref": "#/components/schemas/URL"
                             },
                             "data": {
-                                "type": "string",
-                                "contentEncoding": "base64"
+                                "type": "string"
```

Comment:
Hm, somehow I liked that our docs had this `contentEncoding` explicitly specified here -- we should bring it back. You will need to look into our openapi generator and see how we can annotate the field to bring this back somehow.

Reply:
This comes from if we use the type "bytes" in `data: Optional[bytes]`.

Reply:
@yanxi0830 yeah I see it here https://github.com/meta-llama/llama-stack/blob/main/docs/openapi_generator/strong_typing/schema.py#L267 -- I think we could use some annotation on the str field like for example if we did `data: str = Field(contentEncoding="base64")` and then parse that inside the schema. 

but it's fine to not do it for now. could you just mark a TODO somewhere?

Reply:
modified the generator to make this work